bot/resources/tags/slicing.md

```diff
@@ -0,0 +1,26 @@
+---
+aliases: ["slice", "seqslice", "seqslicing", "sequence-slice", "sequence-slicing"]
+embed:
+    title: "Sequence slicing"
+---
+*Slicing* is a way of accessing a part of a sequence by specifying a start, stop, and step. As with normal indexing, negative numbers can be used to count backwards.
+
+**Examples**
+```py
+>>> letters = ['a', 'b', 'c', 'd', 'e', 'f', 'g']
+>>> letters
+['a', 'b', 'c', 'd', 'e', 'f', 'g']
```

Comment:
Since this provides no new information, it can be omitted.

Reply:
I just removed that.